### DISCUSSION TITLE
FontAwesome V6 Cheatsheet

### DISCUSSION THREAD

**Discussion by AustinAHunter** [FortAwesome/Font-Awesome#20928] [Feature Requests]
The Cheatsheet for V5 of FontAwesome was super helpful when using the icon packs in mobile apps so that we could use a class to translate the unicode for each icon to a more memorable and intellisensed string. See https://fontawesome.com/v5/cheatsheet. Are there plans to release something like this for v6?

**Comment by sensibleworld** [comment]
Hey @AustinAHunter. I'm interested in hearing more about your particular use case. Are you just using the cheatsheet to look-up the unicode values, or are you running some sort of script against it? If the former, we still have those values listed, but you need to open an icon modal to see them.

<img width="1611" alt="CleanShot 2022-03-16 at 08 20 10@2x" src="https://user-images.githubusercontent.com/529604/158588558-2e84710d-62e2-4bca-af4d-c5e56a00cdd9.png">

<img width="1055" alt="CleanShot 2022-03-16 at 08 20 22@2x" src="https://user-images.githubusercontent.com/529604/158588571-eaad997c-e08b-4575-bfce-107d420cd3cb.png">

If the latter, then I might suggest using one of the metadata files in our downloads to access the same info.

**Comment by AustinAHunter** [comment]
Hi @sensibleworld! Thanks for the reply. I was reaching out in regards to another GitHub repo that was using that cheatsheet by scraping the json for the icons to create a more user-friendly, local .cs file that mapped the unicodes to their glyph names. Looking at the metadata file "icons.json", I am hoping that will have all of the information needed to do the same thing, but hopefully easier!

**Comment by tagliala** [comment]
>  local .cs file that mapped the unicodes to their glyph names.

`icons.json` is back in 6.1, but I'm asking if parsing https://github.com/FortAwesome/Font-Awesome/blob/6.x/scss/_variables.scss wouldn't have been easier to associate glyph name to unicode point
